config/develop/bridgeserver2.yaml

```diff
@@ -41,3 +41,7 @@ parameters:
   SynapseOAuthClientId: !ssm /bridgeserver2-develop/SynapseOAuthClientId
   SynapseOAuthClientSecret: !ssm /bridgeserver2-develop/SynapseOAuthClientSecret
   CrcGeoCodeApiKey: !ssm /bridgeserver2-develop/CrcGeoCodeApiKey
+  CuimcTestUsername: !ssm/bridgeserver2-develop/CuimcTestUsername
+  CuimcTestPassword: !ssm/bridgeserver2-develop/CuimcTestPassword
+  CuimcProdUsername: !ssm/bridgeserver2-develop/CuimcProdUsername
+  CuimcProdPassword: !ssm/bridgeserver2-develop/CuimcProdPassword
```

Comment:
If we expect this to be shared across all stacks (dev, staging, prod), we should use bridgeserver2-common instead of bridgeserver2-develop.

Also, I'm not sure, but I think you might need a space between !ssm and the slash.